mkdocs.yml

```
@@ -112,6 +112,7 @@ nav:
     - Appendix:
       - Zero-Trust Network Architecture: appendix/zero-trust.md
       - Ingress DNS setup: appendix/ingress-dns.md
+      - What is production ready: appendix/production-ready.md
```

```suggestion
      - What is production ready?: appendix/production-ready.md
```

tenker vi kan koste på oss et spørsmålstegn her for å ha riktig språksyntaks